### ISSUE TITLE
[FEATURE] Replace open_filex by open_file

### ISSUE THREAD

**Issue by EArminjon** [jhomlala/alice#242]
**Is your feature request related to a problem? Please describe.**
open_filex is a fork of open_file and both package can't live together if i'm not wrong.

**Describe the solution you'd like**
If possible remove open_filex and use open_file. Open_file receive a lot of fix and maybe the usage of the fork is no longer required.

**Describe alternatives you've considered**

**Additional context**

**Comment by techouse** [comment]
@EArminjon hmm, not sure. At the moment of writinf [open_filex](https://pub.dev/packages/open_filex) looks more up to date than [open_file](https://pub.dev/packages/open_file).

**Comment by kostadin-damyanov-prime** [comment]
@EArminjon @techouse While this is true, the downside of using `open_filex` is that it has some permissions in it's Android manifest file that Google Play Store complains about. This prevents us from distributing internal test builds with Alice enabled.

The permissions in question:
```
<uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
<uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
<uses-permission android:name="android.permission.READ_MEDIA_AUDIO" />
```

You can read more about this issue here: https://support.google.com/googleplay/android-developer/answer/14115180?hl=en

So in conclusion - while both packages are maintained I think that `open_file` is preferable because it does have a better approach to handling sensitive permissions.

**Comment by jhomlala** [comment]
Fixed in 1.2.0.
